opennms-doc/guide-admin/src/asciidoc/text/telemetryd/protocols/jti/junos-telemetry-interface.adoc

```diff
@@ -0,0 +1,42 @@
+
```

Comment:
If we move the docs around, we also need to update the index where they are referenced.